src/components/files/FilesOverviewCard/FilesOverviewCard.tsx

```diff
@@ -1,17 +1,20 @@
 import { LibraryBooksOutlined } from '@material-ui/icons';
-import * as React from 'react';
-import { FC } from 'react';
+import React, { FC } from 'react';
 import { FieldOverviewCard } from '../../FieldOverviewCard';
+import { useNumberFormatter } from '../../Formatters';
 
 export interface BudgetOverviewCardProps {
   className?: string;
   loading?: boolean;
+  total: number | undefined;
```

Comment:
```suggestion
  total?: number;
```
Let's just let the prop be omitted